src/Microsoft.XmlSerializer.Generator/src/SgenTasks.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.XmlSerializer.Generator
+{
+    public class SgenParameters : Task
```

Comment:
Maybe I'm missing it, but couldn't you have done the same thing as this with the WriteLinesToFile build in MSBuild task?

Reply:
I didn't know about this task! It is actually very useful in this case.